components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -301,13 +305,46 @@ - (UIFont *)messageFont {
 }
 
 - (void)setBackgroundColor:(UIColor *)backgroundColor {
+  _backgroundColor = backgroundColor;
   self.view.backgroundColor = backgroundColor;
   _tableView.backgroundColor = backgroundColor;
   _header.backgroundColor = backgroundColor;
 }
 
 - (UIColor *)backgroundColor {
-  return self.view.backgroundColor;
+  return _backgroundColor;
```

Comment:
I think it makes sense to leave this as a pass-through to `self.view.backgroundColor`.